### DISCUSSION TITLE
Suggestion: Battery voltage offset in menu

### DISCUSSION THREAD

**Discussion by Overland-DX** [PE5PVB/TEF6686_ESP32#221] [Ideas]
I have noticed that my radio shows lower battery voltage than the battery actually has. For example, the radio shows 3.6V while I measure 3.73v with the measuring device. And when the battery is fully charged, the radio never shows more than 4.0v.
Since all the radios show different values, it would perhaps be nice to be able to adjust this value yourself?

Is it possible/room for an extra digit?
Such as:
3.83V

Perhaps by removing the WI-FI icon since battery and wi-fi can never run at the same time anyway?

**Comment by PE5PVB** [comment]
Main problem is that voltage detector can't be accurate, as I already explained multiple times on discord.
